wiki/start/a-simple-game.md

```diff
@@ -89,11 +89,11 @@ private static Lwjgl3ApplicationConfiguration getDefaultConfiguration() {
 ## Loading Assets
 2D games made in libGDX need assets: images, audio, and other resources that comprise the project. In this case, we'll need a bucket, a raindrop, a background, a water drop sound effect, and music. If you're pretty resourceful, you can make these on your own. For simplicity's sake, you can download these examples which are optimized for this tutorial.
 
-<a href="/assets/downloads/tutorials/simple-game/bucket.png?nomagnify" download="bucket">bucket.png</a><br>
-<a href="/assets/downloads/tutorials/simple-game/drop.png?nomagnify" download="drop">drop.png</a><br>
-<a href="/assets/downloads/tutorials/simple-game/background.png?nomagnify" download="background">background.png</a><br>
-<a href="/assets/downloads/tutorials/simple-game/drop.mp3?nomagnify" download="drop">drop.mp3</a><br>
-<a href="/assets/downloads/tutorials/simple-game/music.mp3?nomagnify" download="music">music.mp3</a>
+<a href="/assets/downloads/tutorials/simple-game/bucket.png?nomagnify" download="bucket.png">bucket.png</a><br>
+<a href="/assets/downloads/tutorials/simple-game/drop.png?nomagnify" download="drop.png">drop.png</a><br>
+<a href="/assets/downloads/tutorials/simple-game/background.png?nomagnify" download="background.png">background.png</a><br>
+<a href="/assets/downloads/tutorials/simple-game/drop.mp3?nomagnify" download="drop.mp3">drop.mp3</a><br>
+<a href="/assets/downloads/tutorials/simple-game/music.mp3?nomagnify" download="music.mp3">music.mp3</a>
```

Comment:
I added extension name to file name so it's downloading with correct extension on ubuntu with firefox. Without this change `mp3` is set as `mpeg`. Could someone with different OS could test that? Also I don't know what that `nomagnify` param means (google either) so I'm leaving it as it is :D